crates/client/l1-messages/src/error.rs

```diff
@@ -0,0 +1,20 @@
+use mp_felt::Felt252WrapperError;
+
+#[warn(dead_code)]
+#[derive(Debug, Eq, PartialEq)]
+pub enum L1MessagesWorkerError {
```

Comment:
It should impl `Error`. Looks like a perfect place to use `thiserror`

Reply:
Done